resources/graphql.raml

```diff
@@ -6,9 +6,10 @@ post:
   securedBy: [oauth_2_0: {scopes: ["manage_project:{projectKey}", "view_products:{projectKey}"]}]
   body:
     application/graphql:
-      example: !include ../examples/graphql-query.example.graphql
```

Comment:
would be good if you can include the example again. In case it's incorrect adjust it please